.github/workflows/flake_finder.yml

```diff
@@ -49,3 +49,11 @@ jobs:
           df -h
           free -h
           make post-mortem
+
+      - name: Raise an issue to report flaky test failure
```

Comment:
I don't know how useful it is in a job that's quite affected by 3rd party factors (image repositories and such).

I fear a flood of automated issues which would just create noise or be ignored since they're automated..

Reply:
Would this be impacted by 3rd party factors we want to ignore often? It doesn't seem like running once a week, and so far always passing, would create much noise.

Although cranking the frequency up to debug a flaky failure, like we're doing in the operator repo currently, definitely does create a lot of noise. I'm okay with removing this step or keeping it, but the feedback previously was to add it.

Reply:
It generates emails so folks are already getting notified that way.  I don't think we need to go a step further and create issues.

Reply:
Okay, so I'll remove it elsewhere too.